doc/nrf-bm/migration/nrf5_bm_migration.rst

```diff
@@ -91,6 +87,22 @@ In |BMshort|, there are a few ready-made partitioning schemes that can be select
 Partitioning can be tweaked by making simple changes to textual **Devicetree** files which define the layout of the memory.
 These can be edited in the board files, or applied to existing boards as **overlays**.
 
+Non-volatile Memory (NVM)
+*************************
+
+The nRF51 and nRF52 devices supported by the nRF5 SDK utilized flash memory as their non-volatile memory (NVM) technology for storing program code and data.
+In contrast, the nRF54L devices employ RRAM, a different NVM technology.
+The inherent differences between flash and RRAM technologies impact the way storage and file systems are implemented.
+
+It is technically feasible to reuse a file system that expects flash behavior on a device that has RRAM.
+This can be done by adding an abstraction layer that emulates the flash behavior.
+However, this approach is generally not recommended for production-ready applications.
+Emulating flash on RRAM can significantly increase the number of erase-write cycles, potentially accelerating wear on the NVM.
```

Comment:
There are no erase cycles on RRAM, so `erase-write cycles` seems a bit wrong. What about:
`Emulating flash memory behavior on RRAM can lead to a significant increase in write operations, potentially accelerating wear on the NVM.`